mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/connectivity/oauth/AuthorizationCode.java

```diff
@@ -73,4 +73,12 @@
    * @since 1.4.0
    */
   CredentialsPlacement credentialsPlacement() default BODY;
+
+  /**
+   * Whether the redirect_uri parameter should be included in the refresh token request. Defaults to {@code true} to avoid
```

Comment:
remove the backwards compatiblity comment. This jdoc is user facing and that's not a comment for that audience